### PR TITLE
Create Works  popup window (Mobile and Desktop)

### PR DESCRIPTION
**Pull request by mzirkof**
### This Request is about adding Pop Up 
⭐ According to the project requirement, We are asked : 
▶️ Use Javascript to Generate the section ' Recent Works ' 👍 
▶️ Use Javascript to Generate details about the works using a Popup Windows 👍  
▶️ A click on the header 'X' button closes the tag 👍 
▶️ Amazing animations ( using @keyFrames , animation property and transition ) added for fun 😆 🤣 🥇  👍 

I hope the 🏈 reaches the 🥅 at a percentage of 💯 
Happy Review their reviewer ☕

### CODE REVIEW

**Review by KanzaTahreem — COMMENTED** [review]
# INVALID ❌ 

Hi @mzirkof  👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😓

## Invalid Code Review Request ❌
- [ ] Unfortunately, you have submitted a pull request that lacks commits from your coding partner(s). Please note that this is a pair programming activity, and everyone should contribute to the project ⛏🤓
- [ ] Please make sure that all headers are present in README, the only optional section is FAQ 👩‍⚖️

_**Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready**_ 🤝🙏

Cheers and Happy coding!👏👨‍💻💻

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by mzirkof** [comment]
@KanzaTahreem  Hi Madam The fact is that due to an issue with my coding partner student success team told me to work alone which is why there is no commit from a coding partner. I work alone unfortunately

**Comment by mzirkof** [comment]
@KanzaTahreem a capture : 
 
Capture
 
I hope i am not losing a review

**Comment by KanzaTahreem** [comment]
Hello @mzirkof, 
Got it. If you face a similar situation next time, please attach a similar screenshot for CRs so that they can review your project. Please go ahead and ask for a review. I wish you success 🥇
**_Note: An Invalid Code Review Request does not count towards the code review limit._**
If you have any further questions, feel free to ask!

File: styles/style.css [hunk 1]
```diff
@@ -1098,12 +1100,129 @@
 
   .pop_up .container {
```
Comment (mwanawabangona) [inline styles/style.css:1101]: - [ ] `required`
- [ ] God job with the pop-up so far!! 👏 
- [ ] However, i noticed you did not implement the background blur on both `mobile` and `desktop`
- [ ] Please implement 🥂 
- [ ] please see screenshot comparison

`your template`
image

`original template`
image

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @mzirkof ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi mzirkof, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job implementing the changes requested by the previous CR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**